### PR TITLE
Feat: Add Json schema on execute page

### PR DESCRIPTION
**Pull request by evilpeach**
## Describe your changes
- add JSON schema on execute page

### CODE REVIEW

**Review by poomthiti — APPROVED** [review]

File: src/lib/pages/execute/components/ExecuteArea.tsx [hunk 1]
```diff
@@ -1,304 +1,81 @@
...
-      <Flex alignItems="center" justify="space-between">
-        <Flex gap={2}>
-          <CopyButton
-            isDisable={!msg.length}
-            value={msg}
-            amptrackSection="execute_msg"
-          />
-          <CodeSnippet
-            type="execute"
+      <MessageInputContent
```
Comment (songwongtp) [inline src/lib/pages/execute/components/ExecuteArea.tsx:57]: could you help change `MessageInputContent` to use implicit return

File: src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    msg,
+    assetsJsonStr,
+    assetsSelectString,
+    getAttachFunds,
+    attachFundsOption,
+    assetsSelect,
+  ]);
+
+  return (
+    <AccordionItem p={4}>
```
Comment (songwongtp) [inline src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx:217]: remove `p={4}`

File: src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+      <h6>
+        <AccordionButton>
```
Comment (songwongtp) [inline src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx:219]: add `p={4}` so user can click the whole accordion

File: src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+          <Box w="full" textAlign="start">
+            <Text variant="body1" fontWeight={700}>
+              {msgSchema.title}
+            </Text>
+            <Text variant="body1">{msgSchema.description}</Text>
+          </Box>
+          <AccordionIcon />
+        </AccordionButton>
+      </h6>
+      <AccordionPanel p="16px 0 0 0">
```
Comment (songwongtp) [inline src/lib/pages/execute/components/schema-execute/ExecuteBox.tsx:229]: change `p` to `p={4}`

**Review by songwongtp — APPROVED** [review]

**Review by poomthiti — APPROVED** [review]

**Pull request merged**